### PR TITLE
attaching message context to handler invocations

### PR DESCRIPTION
**Pull request by cdm-ium**
Pubsub equivalent of: https://github.com/globality-corp/microcosm-flask/pull/51

### CODE REVIEW

File: microcosm_pubsub/dispatcher.py [hunk 1]
```diff
@@ -78,4 +91,6 @@ def configure_sqs_message_dispatcher(graph):
     return SQSMessageDispatcher(
         sqs_consumer=graph.sqs_consumer,
         sqs_message_handlers=sqs_message_handlers,
+        log_with_context=graph.config.sqs_message_dispatcher.log_with_context,
```
Comment (jessemyers) [inline microcosm_pubsub/dispatcher.py]: Seems like if you set log with context to False, you are still required to wire in `graph.message_context`?

File: microcosm_pubsub/dispatcher.py [hunk 2]
```diff
@@ -78,4 +91,6 @@ def configure_sqs_message_dispatcher(graph):
...
+        message_context=graph.message_context,
```
Comment (jessemyers) [inline microcosm_pubsub/dispatcher.py]: How about `sqs_message_context`?

**Comment by jessemyers** [comment]
LGTM if you address the usability/configurability issues.

File: microcosm_pubsub/dispatcher.py [hunk 3]
```diff
@@ -75,7 +83,15 @@ def configure_sqs_message_dispatcher(graph):
     except NotBoundError:
         sqs_message_handlers = graph.config.sqs_message_dispatcher.mappings
 
+    try:
```
Comment (cdm-ium) [inline microcosm_pubsub/dispatcher.py]: I removed the `log_with_context` flag in favor of having a noop default that we can always use. User can have more fine-grained control by binding the key `sqs_message_context` with either the default function from this repo or their own custom context.

**Pull request closed without merging**